Mathlib/Data/DFinsupp/Order.lean

```diff
@@ -40,16 +40,15 @@ variable [∀ i, LE (α i)] {f g : Π₀ i, α i}
 instance : LE (Π₀ i, α i) :=
   ⟨fun f g ↦ ∀ i, f i ≤ g i⟩
 
+instance : DFunLike.PointwiseLE (Π₀ i, α i) where
+
 lemma le_def : f ≤ g ↔ ∀ i, f i ≤ g i := Iff.rfl
 #align dfinsupp.le_def DFinsupp.le_def
 
-@[simp, norm_cast] lemma coe_le_coe : ⇑f ≤ g ↔ f ≤ g := Iff.rfl
+lemma coe_le_coe : ⇑f ≤ g ↔ f ≤ g := Iff.rfl
```

Comment:
Shouldn't that stay a dsimp lemma?

Reply:
`Iff.rfl` lemmas aren't `dsimp` lemmas.

Reply:
Hmm, in Lean 3 they were

Reply:
Can you leave a porting note about how this should be a dsimp lemma but currently is not?